gen/generate.go

```diff
@@ -326,13 +340,18 @@ func generateGenesisFiles(outDir string, protoVersion protocol.ConsensusVersion,
 		fmt.Fprintln(verboseOut, protoVersion, protoParams.MinBalance)
 	}
 
+	if rewardsBalance < protoParams.MinBalance {
+		// Needs to at least have min balance
+		rewardsBalance += protoParams.MinBalance - rewardsBalance
```

Comment:
@barnjamin Sanity checking my intuition:  Is there a reason to favor the PR vs the _suggestion_?

I'm trying to gauge in what cases it's preferable to allocate > min balance, rather than coercing to min balance.

```suggestion
		rewardsBalance = protoParams.MinBalance
```